internal/cloud/realm/import_export_test.go

```diff
@@ -246,7 +246,7 @@ console.log('got heem!');
 				"disabled":      false,
 			},
 		},
-		GraphQL: local.GraphQLStructure{
+		GraphQL: &local.GraphQLStructure{
```

Comment:
Made `V1.GraphQL` consistent with all other `*Structure`s.

Reply:
the pointer thing isn't supposed to apply everywhere... that was more when it was needed to understand whether something was present or not, and with v1 graphql was _always present

you can revert this